pkg/helper/client.go

```diff
@@ -80,6 +80,28 @@ func set(state string) (string, error) {
 	return stdout.String(), nil
 }
 
+func Rollback() (string, error) {
+	cmd := exec.Command(nmstateCommand, "rollback")
```

Comment:
Could you  create a helper function to run nmstatectl commands? This is being duplicated